nodetool

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+# If STARGATE_HOME wasn't specified in the environment, then search for one...
+if [ "x$STARGATE_HOME" == "x" ]; then
+    STARGATE_HOME=`dirname "$0"`
+    if [[ ! -d $STARGATE_HOME ]]; then
+      echo "STARGATE_HOME not found"
+      exit 2
+    fi
+else
+    if [[ ! -d $STARGATE_HOME ]]; then
+      echo "STARGATE_HOME: '$STARGATE_HOME' is not a directory"
+      exit 2
+    fi
+fi
+
+declare -a CMD=("${STARGATE_HOME}/starctl" "--nodetool")
+
+# If the Stargate LISTEN address is set use it for the --host nodetool option by default
+if [[ ! -z "$LISTEN" ]]; then
+    CMD+=("-T" "--host" "-T" "$LISTEN")
+fi
+
+# Build starctl options to forward cmd line args to nodetool
+for a in "$@" ; do
+  CMD+=("-T" "$a")
+done
+
+# Use environment variables to determine persistence type and version
+if [[ ! -z "$DSE" ]]; then
+    CMD+=("--dse")
+fi
+
+if [[ ! -z "$CLUSTER_VERSION" ]]; then
+    CMD+=("--cluster-version" "$CLUSTER_VERSION")
+fi
```

Comment:
Is this the only config needed? If so does it make sense to reuse starctl and Starter.java or would it be cleaner to introduce a new Nodetool.java?

Reply:
We only need those options that are required for selecting the persistence bundle (I guess it makes sense to use the same `NodeTool` version as the backend).

Adding a new command class seemed too invasive to me, mainly because `rvesse.airline` does not have an easy way to group common options (at least I could not find it). All options have to be declared on the class hierarchry, which makes extending harder - we'd have to duplicate all command classes wherever they are extended.